h_periodic/h_beat.py

```diff
@@ -55,8 +55,8 @@
         "fill-annotations-pk": {
             "options": {"expires": 30},
             "task": "h.tasks.annotations.fill_pk_and_user_id",
-            "schedule": crontab(hour="*,9-12", minute="*/15"),
-            "kwargs": {"batch_size": 10000},
+            "schedule": crontab(hour="*,6-12", minute="*/10"),
+            "kwargs": {"batch_size": 15000},
```

Comment:
Maybe more, 50k?

Reply:
How long is this going to take to complete given the current DB size and growth rate?

Reply:
With this rate over two months, new annotations are not that important, we can (and should) soon update the code so they don't need this, setting the pk and user_id in python when they are created.